framework/rest/Serializer.php

```diff
@@ -152,6 +152,11 @@ public function serialize($data)
             return $this->serializeModel($data);
         } elseif ($data instanceof DataProviderInterface) {
             return $this->serializeDataProvider($data);
+        } elseif (is_array($data)) {
+            $serializedArray = [];
+            foreach ($data as $key => $value)
```

Comment:
`{` and `}` should be added for foreach.

Reply:
I've done both. You can find everything in this branch